js/localgov_eu_cookie_compliance.es6.js

```diff
@@ -103,10 +103,24 @@
    * Provide feedback when the "Save cookie settings" button is clicked.
    */
   function setupSaveSettingsFeedback() {
-    $(".eu-cookie-compliance-save-preferences-button").click(e => {
-      $(`<p>${Drupal.t("Saved")}</p>`)
-        .appendTo(e.target)
-        .hide(2000);
-    });
+    var saveButton = document.querySelector('.eu-cookie-compliance-save-preferences-button');
+    const saveButtonLabel = saveButton.innerHTML;
+
+    saveButton.addEventListener("click", function () {
+      saveButton.innerHTML = `${Drupal.t("Saving")}`;
+      saveButton.classList.add('spinning');
+
+      setTimeout(
+        function () {
+          saveButton.classList.remove('spinning');
+          saveButton.innerHTML = `${Drupal.t("Preferences saved")}  <i class="fas fa-check"></i>`;
```

Comment:
I'm nervous of these classes. Not all themes will have font-awesome installed. I'd guess most themes will not have it if they are sub themes of Localgov Base which I think most councils are using for their base theme.

We should rewrite these using BEM-style classes. I also have [a larger issue](https://github.com/localgovdrupal/localgov_eu_cookie_compliance/issues/2) to do that for this whole module.

Reply:
I'd initially filtered these out of the PR compared to what we were doing internally, but then noticed that the module as a whole was using them. I think consistency is really important here, so if #2 is broadly acceptable then we can aim this for the same release that makes that change. I'd feel a bit strange about removing these while the core templates include the checks.

Reply:
Let's leave them in for now, and we'll fix all those classes on one go via #2 when we get around to it.

It might mean leaving in the bootstrap classes so anyone currently using this module won't have a breaking change and additionally adding our new BEM classes.